default_cluster_issuer.tf

```diff
@@ -0,0 +1,79 @@
+locals {
+  cluster_issuers_values = yamlencode({
+    "ingressShim" : {
+      "defaultIssuerName" : "default"
+      "defaultIssuerKind" : "ClusterIssuer"
+      "defaultIssuerGroup" : "cert-manager.io"
+    }
+  })
+}
+
+data "utils_deep_merge_yaml" "default_cluster_issuer_values" {
+  count = var.enabled ? 1 : 0
+  input = compact([
+    var.cluster_issuers_values
+  ])
+}
+
+resource "helm_release" "default_cluster_issuer" {
+  count = var.enabled && var.argo_enabled && !var.argo_helm_enabled && var.cluster_issuer_enabled ? 1 : 0
```

Comment:
Why is  `!var.argo_helm_enabled` used here? This resource should be created when we don't use ArgoCD so this if should be structured differently

Reply:
What happens when ArgoCD is not used at all? I don't think any issuer can be created that way.